server/app/views/admin/programs/ProgramIndexViewV2.java

```diff
@@ -4,18 +4,19 @@
 import static j2html.TagCreator.div;
 import static j2html.TagCreator.each;
 import static j2html.TagCreator.h1;
-import static j2html.TagCreator.input;
-import static j2html.TagCreator.label;
 import static j2html.TagCreator.p;
+import static j2html.TagCreator.span;
```

Comment:
Note to the reviewer: The review flow collapsed this by default for me. Don't forget this file since this is where 99% of the changes are :).

Reply:
I really wish github didn't do this :(

Reply:
Yah, I found it pretty surprising. It seems that collapsing makes it possible to hide the changes that actually matter. I'd be more comfortable if this was applied to binary-type files rather than text files.